### ISSUE TITLE
AssertionError Boxes

### ISSUE THREAD

**Issue by mdaehl** [megvii-research/Iter-E2EDET#29]
Hello, I like your work and wanted to try your code, but I am constantly encountering AssertionErrors (see image). I use the config you recommend for 1 GPU: "--config-file configs/50e.6h.500pro.ignore.yaml --num-gpus 1 SOLVER.IMS_PER_BATCH 2 SOLVER.BASE_LR 0.0025" and crowdhuman data.  After debugging I realized that the network predicts nan boxes that cause the error. However, I do not know how to cirumvent this. In your code I found the comment "degenerate boxes gives inf / nan results", but there is no indication how to fix this. Do you have an idea how to solve this problem or how did you handle such boxes? (Note: The loss is decreasing during the training.)

Thanks in advance!

<img width="884" alt="crash" src="https://github.com/megvii-research/Iter-E2EDET/assets/57526167/44072d03-12d0-4fff-8c39-6b880721f0d4">

**Comment by yexiguafuqihao** [comment]
Try to decrease the learning rate. One case is to replace the learning rate with 0 and observe whether the mistake happens or not.
